.stylelintignore

```diff
@@ -0,0 +1 @@
+css/semantic.min.css
```

Comment:
Add an empty line at the end of the file.  (You can set your editor to do this for you)